lib/App/GitHooks.pm

```diff
@@ -452,6 +452,8 @@ sub run
 		croak "Invalid hook name $name"
 			if scalar( grep { $_ eq $name } @$HOOK_NAMES ) == 0;
 
+		return $HOOK_EXIT_SUCCESS if _should_skip( $name );
```

Comment:
Yes. We should also create a follow-up ticket to add a note to the commit message if the pre-commit hook was skipped.